docker-compose.yml

```
@@ -0,0 +1,20 @@
+services:
+  gitlab:
+    image: gitlab/gitlab-ce:17.4.2-ce.0
+    container_name: pippo
+    hostname: pippo
+    restart: unless-stopped
+    environment:
+      https: false
+      GITLAB_OMNIBUS_CONFIG: |
+        external_url 
```

yamllint

[error] trailing spaces

(trailing-spaces)